astropy/utils/iers/data/update_builtin_iers.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Oh, I now wonder whether a better place for this script would be in the `astropy-procedures` repo, rather than in core?